integr8ly/integreatly-operator
# Cro resources dashboard

# Description

Added new Grafana dashboard that displays CRO resource metrics

![image](https://user-images.githubusercontent.com/45426048/109139546-8ad2e500-7753-11eb-91ae-af41c007fd08.png)

## Type of change




## Checklist

## Verification
- Clone this branch
- Deploy locally on byoc cluster
- Check if CRO resources dashboard exists in middleware monitoring operator grafana
- Verify that the dashboard is displaying data
- Changed is deployed here (mwcollabaz-byoc) to verify visually
